### PR TITLE
Bump Librsvg to v2.52.4

### PR DESCRIPTION
**Pull request by originalsouth**
Attempt to update Librsvg in order to resolve https://github.com/MichaelHatherly/MathJaxRenderer.jl/issues/9

### CODE REVIEW

**Comment by originalsouth** [comment]
Seems broken for now, rustc version is too old (librsvg needs at least rustc version 1.52)...
Probably related too https://github.com/JuliaPackaging/Yggdrasil/issues/3618

File: L/Librsvg/build_tarballs.jl [hunk 1]
```diff
@@ -3,65 +3,20 @@
...
-fi
-
-# Manually build rust library because rust target doesn't match C target
-(cd rust && PKG_CONFIG_ALLOW_CROSS=1 cargo build --release)
-
-RUST_LIB="$(pwd)/rust/target/${rust_target}/release/librsvg_internals.a"
-make RUST_LIB="${RUST_LIB}" -j${nproc}
-make RUST_LIB="${RUST_LIB}" install
+./configure --host=${target} --build=${MACHTYPE} --prefix=${prefix} --disable-static --enable-pixbuf-loader --disable-introspection --disable-gtk-doc-html --enable-shared
+make
```
Comment (giordano) [inline L/Librsvg/build_tarballs.jl:19]:
```suggestion
./configure --host=${rust_target} \
    --build=${MACHTYPE} \
    --prefix=${prefix} \
    --disable-static \
    --enable-pixbuf-loader \
    --disable-introspection \
    --disable-gtk-doc-html \
    --enable-shared
make
make install
```

**Comment by giordano** [comment]
Let's see how much is still broken :smile:  Yesterday I had troubles on musl and aarch64-apple-darwin, it worked on x86_64-linug-gnu, but didn't try much more else.

**Comment by giordano** [comment]
Eight platforms actually successful, that's a progress! :partying_face:

**Comment by giordano** [comment]
We're down to 2 failing platforms!

**Comment by giordano** [comment]
SUCCESS! :partying_face:

**Comment by giordano** [comment]
Alright, I'm going ahead and merge this.  Thanks for starting this out, this was the final motivation for me to resuscitate the Rust toolchain, which is now in a much better shape! :slightly_smiling_face:

**Comment by originalsouth** [comment]
Awesome! Thank you so much for finishing this, great work!

**Pull request merged**